### ISSUE TITLE
federated SVR model

### ISSUE THREAD

**Issue by 9527-csroad** [adap/flower#1703]
### What is your question?

Hi, thans for job, it's meaningful.
Now, I want to use sklearn and flwr to train a SVR model, and some questions happend.
Plz look my revise ,this is the utils.py
![image](https://user-images.githubusercontent.com/52602255/223600869-28cc17aa-623d-497f-ba67-69f2de0f13d9.png)
this is my client
`
if __name__ == "__main__":
    train_data = pd.read_csv("data/bostonp.csv")
    x_train = train_data.iloc[0:159, :-1].values
    y_train = train_data.iloc[0:159, -1].values
    test_data = pd.read_csv("data/bostonp.csv")
    x_test = test_data.iloc[160:169, :-1].values
    y_test = test_data.iloc[160:169, -1].values

    model = SVR()
    model_data = pd.read_csv("data/bostonp.csv")
    model_x = model_data.iloc[0:19, :-1].values
    model_y = model_data.iloc[0:19, -1].values
    model = model.fit(model_x, model_y)

    class Client(fl.client.NumPyClient):
        def get_parameters(self, config):  # type: ignore
            return utils.get_model_parameters(model)

        def fit(self, parameters, config):  # type: ignore
            utils.set_model_params(model, parameters)
            # Ignore convergence failure due to low local epochs
            with warnings.catch_warnings():
                warnings.simplefilter("ignore")
                model.fit(x_train, y_train)
            print(f"Training finished for round {config['server_round']}")
            return utils.get_model_parameters(model), len(x_train), {}

        def evaluate(self, parameters, config):  # type: ignore
            utils.set_model_params(model, parameters)
            loss = r2_score(y_test, model.predict(x_test))
            accuracy = model.score(x_test, y_test)
            return loss, len(x_test), {"accuracy": accuracy}

    fl.client.start_numpy_client(server_address="192.168.1.13:8888", client=Client())
`
and this is my server
`
def fit_round(server_round: int) -> dict:
    return {"server_round": server_round}

def get_evaluate_fn(model: SVR):
    test_data1 = pd.read_csv("data/bostonp.csv")
    x_test = test_data1.iloc[490:506, :-1].values
    y_test = test_data1.iloc[490:506, -1].values  # replace with your own test labels

    def evaluate(server_round, parameters: fl.common.NDArrays, config):
        # Update model with the latest parameters
        utils.set_model_params(model, parameters)
        loss = mean_squared_error(y_test, model.predict(x_test))
        # Predict on test set
        accuracy = model.score(x_test, y_test)
        return loss, {"accuracy": accuracy}

    return evaluate

if __name__ == "__main__":
    model = SVR()
    model_data = pd.read_csv("data/bostonp.csv")
    model_x = model_data.iloc[0:19, :-1].values
    model_y = model_data.iloc[0:19, -1].values
    model = model.fit(model_x, model_y)
    strategy = fl.server.strategy.FedAvg(
        min_available_clients=3,
        evaluate_fn=get_evaluate_fn(model),
        on_fit_config_fn=fit_round,
    )
    fl.server.start_server(
        server_address="192.168.1.13:8888",
        strategy=strategy,
        config=fl.server.ServerConfig(num_rounds=5),
    )
`
I have been tried once, it worked. I have been checked the code and dont found error.
But when i try the new data, it's wrong
![image](https://user-images.githubusercontent.com/52602255/223601340-75bfd494-fd50-4fd2-a99b-9828c54801b9.png)
plz tell me where is my fault, thanks

**Comment by WilliamLindskog** [comment]
Hi @9527-csroad,

is this an issue that you are still experiencing?

**Comment by WilliamLindskog** [comment]
Hi again, 

The current sklearn tabular example can be modified to create a regression task using SVR. Thus, closing this.
